src/view/view-layout.js

```diff
@@ -0,0 +1,75 @@
+import _ from 'underscore';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import Sink from './sink';
+import sinkLayoutGen from './sink-layout-gen';
+
+class View extends Component {
```

Comment:
I see, can you pass the `job_id` as a `key` prop to ensure `ViewLayout` gets unmounted every time the `job_id` changes?